AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java

```diff
@@ -60,7 +60,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Create the catalog object which will start and play the video.
         Catalog catalog = new Catalog("FqicLlYykdimMML7pj65Gi8IHl8EVReWMJh6rLDcTjTMqdb5ay_xFA..");
```

Comment:
Can you give me a reference that I can examine to see if it is what I am looking for?